pkg/option/config.go

```diff
@@ -1963,6 +1971,8 @@ var (
 
 		k8sEnableLeasesFallbackDiscovery: defaults.K8sEnableLeasesFallbackDiscovery,
 		APIRateLimit:                     make(map[string]string),
+
+		EnableClusterIPExternalAccess: defaults.EnableClusterIPExternalAccess,
```

Comment:
Yep it isn't strictly necessary (as it'll default to false if unspecified). I added it since there were bunch of other boolean options that defaulted to false as well and were specified here. I'm fine either way.